commands.go

```diff
@@ -183,6 +201,92 @@ func (cmd *MonitorStatus) SetValue(result *Result) error {
 	return nil
 }
 
+// FreezeFrame represents a command that checks the DTC that caused required 
+// freeze frame data storage
```

Comment:
Since the [mode is hard coded as `0x01`](https://github.com/rzetterberg/elmobd/blob/b4758b44f535d4ed57e51fac14231f218eb88caf/commands.go#L38), when run, this command actually runs the command "Freeze DTC" and not "DTC that caused freeze frame to be stored".